src/service/service.hashtag.js

```diff
@@ -15,7 +17,9 @@ const makeConfig = (token) => {
 
 const getHashtagTrending = (token) => {
 	const promise = axios.get(`${API_HASHTAG}/trending`, makeConfig(token));
-	console.log(promise);
+	if (isDeveloping) {
+		console.log(promise);
+	}
```

Comment:
E aqui :).

Reply:
Na verdade foi proposital, porque eu vou usar esse mesmo arquivo na page da hashtag específica, então eu vou querer ficar habilitando e desabilitando isso aí volta e meia.
Daí eu pensei em deixar assim porque quem for usar componentes que envolvam esses meninos, podem facilmente só trocar o true para false, daí (na minha cabeça) não ia dar tanto problema com conflito.

Seria uma melhor ideia só tirar eles de uma vez?

Reply:
Beleza, então, só não esquece de no final não deixar o console log pra o usuário ver